### PR TITLE
Release/v0.1.1

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**
### Changed

- **\[BREAKING\]** Fixed method name `addResourceToWhitelist`
- **\[BREAKING\]** Fixed method name `removeResourceFromWhitelist`
- Throw early when params are missing in scripts
- Fixed TS typing errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
 # Changelog
 
+## [0.1.1]
+
+### Changed
+
+- **\[BREAKING\]** Fixed method name `addResourceToWhitelist`
+- **\[BREAKING\]** Fixed method name `removeResourceFromWhitelist`
+- Throw early when params are missing in scripts
```
Comment (Le-Caignec) [inline CHANGELOG.md:9]:
```suggestion
- Throw an issue earlier when params are missing in scripts
```

File: package.json [hunk 2]
```diff
@@ -1,45 +1,41 @@
 {
   "name": "whitelist-smart-contract",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (Le-Caignec) [inline package.json:3]: breaking change should be a major version no ? => 0.2.0

**Review by Le-Caignec — CHANGES_REQUESTED** [review]
juste some changes

**Comment by PierreJeanjacquot** [comment]
opened https://github.com/iExecBlockchainComputing/whitelist-smart-contract/pull/6

**Pull request closed without merging**